elado/vonyarcmarina/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: elado
+title: Eladó egy fejlesztés alatt álló új, balatoni kikötő
```

Comment:
Tudom, de a képekről is kitalálható annak, aki ismeri a Balatont... Persze, ha már felhívtad a figyelmemet, lehet, hogy a vonyarcot kihagynám az URL-ből.